README.md

```diff
@@ -387,6 +387,13 @@ The IP configuration supports the following options:
     When using a caching DNS plugin (dnsmasq or systemd-resolved in NetworkManager.conf)
     then "edns0" and "trust-ad" are automatically added.
 
+- `gateway4` and `gateway6`
+
+    The gateway associated with the IPv4 configuration and IPv6 configuration. This is
+    only meaningful if the corresponding IPv4 addresses or IPv6 addresses is also set.
```

Comment:
This sentence is not grammatically correct since addresses is plural and "is" is singular.
Also, I guess the gateway is still useful when using any auto configuration or with only a single address.

Reply:
The reason that I considered IPv4 addresses as singular is because in NM `addresses`  is a property[1]. Maybe I should quote the "addresses"( IPv4 "addresses" ):

![image](https://user-images.githubusercontent.com/36004580/114708299-29fa6e80-9cf9-11eb-8cd1-917d682b7fe2.png)

[1] https://developer.gnome.org/NetworkManager/stable/nm-settings-nmcli.html

Reply:
if you mean the property, a possible way to write it is: Specifying a gateway only works if the IPv6 `addresses` property is set.

On a technical level, I guess setting a gateway and enabling DHCP might also be correct but I might be wrong.